mail/lbdb/Portfile

```diff
@@ -51,7 +52,7 @@ post-destroot {
 }
 
 variant gpg conflicts gpg2 description {Add support for GnuPG version 1} {
-    depends_lib-append      port:gnupg
+    depends_lib-append      port:gnupg1
     configure.args-replace  --without-gpg --with-gpg
 }
 
```

Comment:
Down below in gpg2 variant, `--with-gpg=${prefix}/bin/gpg2` should be changed to `--with-gpg=${prefix}/bin/gpg` like other ports.

P.S. GitHub doesn't allow me to directly comment there.

Reply:
>Down below in gpg2 variant, --with-gpg=${prefix}/bin/gpg2 should be changed to --with-gpg=${prefix}/bin/gpg like other ports.

Nope, because I want the `gpg2` variant to explicitly depend upon a modern GnuPG port, not to be also satisfied by `gnupg1`. This, for instance, is also true for the `gpgme` port (which is a useful wrapper for modern GnuPG.)
I could hardcode `gnupg2` here, but I'd like to keep it a `path:`-based dependency in case we want to add a `gnupg-devel` port in the future for testing versions.

> P.S. GitHub doesn't allow me to directly comment there.

You can only add comments in or around changed hunks, unfortunately. I usually also do what you did - add a comment as close to the location I intended to when I hit this problem.

Reply:
`depends_lib` should remain as is so it explicitly depends upon a modern GnuPG port, I'm just saying that the symlink is only there for compatibility so we should avoid calling it and change `configure.args`. `gnupg2` would conflict with any port that contains `${prefix}/bin/gpg` so this shouldn't be a problem.

Reply:
Ah, you mean the configure argument. Yeah, that was how I implemented it initially, but I don't plan to drop the `gpg2` symlink any time soon and I'd rather see a build failure and a runtime failure if users decide to forcefully deactivate the modern GnuPG version and install the legacy one than it tagging along - potentially in a faulty way.

After thinking that through, I reverted back to the current implementation which feels safer to me.